CHANGES.rst

```diff
@@ -53,6 +53,9 @@ resample
   image shape and the bounding box both for ``SCI`` image as well as for the
   ``ERR`` and ``VARIANCE_*`` images. [#7774]
 
+- Update the following exposure time keywords: XPOSURE (EFFEXPTM), EFFINTTM,
```

Comment:
EFFINTTM should not be included in the updates. It's integration time within a single exposure and doesn't change for combined products.